src/test/java/racingcar/model/car/CarsTest.java

```diff
@@ -0,0 +1,66 @@
+package racingcar.model.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.TestDataManager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarsTest {
+    @DisplayName("자동차 전진 테스트")
+    @Test
+    void movingCarsTest() {
+        // given
+        Cars cars = TestDataManager.getCarsTestData();
+
+        // when
+        for (int i = 0; i < 2; i++) {
+            cars.moveCars();
+        }
+        String carsPositionFormat = cars.getCarsPositionFormat();
+
+        // then
+        assertThat(carsPositionFormat)
+                .contains("pobi : ---")
+                .contains("crong : -");
```

Comment:
`.contains`는 `CharSequence...` 타입으로 받고 있어서 ,로 연결시켜줘도 됩니다!
```suggestion
        assertThat(carsPositionFormat)
                .contains("pobi : ---", "crong : -");
```